### ISSUE TITLE
Install-solc script depends on pipenv

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#864]

**Comment by cnusrini** [comment]
waiting for this feature to be available

**Comment by KPrasch** [comment]
As a temporary work-around - If you want to develop NuCypher without `pipenv` you can install the solidity compiler manually, and place it in your virtual env's path.

For example:
```
wget "https://github.com/ethereum/solidity/releases/download/v0.5.3/solc-static-linux" -O venv/bin/solc
chmod +x venv/bin/solc
```

**Comment by michwill** [comment]
How about determining the `bin` directory to put solc into by:
```
dirname `which python`
```

**Comment by vzotova** [comment]
this issue was solved in #1296 and after #1627 install_solc script depends only on `nucypher/blockchain/eth/sol/__init__.py` from repo
